src/Sparrow/Json/BlittableWriter.cs

```diff
@@ -494,8 +613,12 @@ private static int CalculateCompressionAcceleration(int size)
         public void Dispose()
         {
             _unmanagedWriteBuffer.Dispose();
+
             if (_compressionBuffer!= null)
                 _context.ReturnMemory(_compressionBuffer);
+            if (_allocated != null )
```

Comment:
After returning the memory, need to set `_allocated` to null. This object may be disposed multiple times.